app/ports/templates/ports/port-detail/installation_stats.html

```diff
@@ -37,7 +37,7 @@ <h4 class="border-bottom">Statistics: <span style="font-size:12px"><button type=
                 </tr>
                 <tr>
                     <th scope="row">Most Installed Version</th>
-                    <td><strong>{{ port_installations_by_port_version.first.version }}</strong> : {{ port_installations_by_port_version.first.num }} user(s)</td>
```

Comment:
Why did this stop working in the first place?

Reply:
There were changes done to the `port_installations_by_port_version` query set:
1. Version are no longer sorted by number of users, so first object will not give the most installed version.
2. It is no longer a query set, but a list of dictionaries.